src/interactive/authoring.tsx

```diff
@@ -1,24 +1,49 @@
 import * as React from "react";
 import { IAuthoringInitInteractive, useAuthoredState } from "@concord-consortium/lara-interactive-api";
-
+import { RichTextWidget } from "../components/rich-text-widget";
 import { defaultAuthoredState, IAuthoredState } from "./types";
 
 interface Props {
   initMessage: IAuthoringInitInteractive<IAuthoredState>;
 }
 
 export const AuthoringComponent: React.FC<Props> = ({initMessage}) => {
-  const {authoredState, setAuthoredState} = useAuthoredState<IAuthoredState>();
-  const { useFakeSensor } = authoredState || defaultAuthoredState;
+  const {authoredState, setAuthoredState } = useAuthoredState<IAuthoredState>();
+  const { useFakeSensor, prompt, hint } = authoredState || defaultAuthoredState;
+
+  const updateAuthoredState = (newState: Partial<IAuthoredState>) => {
```

Comment:
Funny - we wrote the same function with different implementations on the same day.  I'm thinking we should update `useAuthoredState` and `useInteractiveState` in the Lara client api to also return update functions.